wit/blob-store.wit

```diff
@@ -0,0 +1,103 @@
+// wasi-blob-store based on https://github.com/WebAssembly/wasi-blob-store
+
+use { container-metadata, object-name, object-metadata, object-id } from blob-types
+
+// a Container is a collection of objects
+resource container {
+	static open: func(name: string) -> expected<container, error>
+
+	// returns container name
+	name: func() -> expected<string, error>
+  
+	// returns container metadata
+	info: func() -> expected<container-metadata, error>
+  
+	// begins reading an object
+	read-object: func(name: object-name) -> expected<read-stream, error>
+  
+	// creates or replaces an object.
+	write-object: func(name: object-name) -> expected<write-stream, error>
+	
+	// retrieves an object or portion of an object, as a resource.
+	// Start and end offsets are inclusive.
+	// Once a data-blob resource has been created, the underlying bytes are held by the blobstore service for the lifetime
+	// of the data-blob resource, even if the object they came from is later deleted.
+	// get-data: func(name: object-name, start: u64, end: u64) -> expected<data-blob, error>
```

Comment:
Was this purposefully commented out?

Reply:
Yes, it was purposefully commented out and I will write a comment here expanding the reason. This PR is not going to implement the `data-blob` resource as I am not sure how it will add value in addition to the `write-stream` resource which is implemented.